crates/core/src/db/datastore/locking_tx_datastore/mod.rs

```diff
@@ -2097,12 +2082,12 @@ impl MutTxDatastore for Locking {
     }
 
     fn get_mut_tx<'a>(
-        &'a self,
+        &self,
         tx: &'a Self::MutTxId,
         table_id: TableId,
-        row_id: Self::RowId,
-    ) -> super::Result<Option<Self::DataRef>> {
-        tx.lock.get(&table_id, &row_id)
+        row_id: &'a Self::RowId,
```

Comment:
This sort of change in lifetimes here is really the heart of this change, I think. In a mem-arch'd future, is this the one you want?

Reply:
Yeah, this comment wasn't intended to be blocking at all and I certainly think iterating fast is the way to go now.